display_information/show-labels-on-layer/README.md

```diff
@@ -1,24 +1,43 @@
-# Show Labels on Layer
+# Show labels on layers
 
-Add custom labels to a layer.
+Display custom labels on a feature layer.
 
-![](ShowLabelsOnLayer.png)
+![Image of show labels on layers](ShowLabelsOnLayer.png)
 
-## How it works
+## Use case
+
+Labeling features is useful to visually display a key piece of information or attribute of a feature on a map. For example, you may want to label rivers or street with their names. 
+
+## How to use the sample
 
-To show labels on a feature layer:
+Pan and zoom around the United States. Labels for congressional districts will be shown in red for Republican districts and blue for Democrat districts. Notice how labels pop into view as you zoom in.
 
-1. First, create a `FeatureLayer` with a `ServiceFeatureTable` using an online feature service.
-2. Create a `TextSymbol` to use for displaying the label text.
-3. Create a JSON string for the label definition.
+## How it works
+
+1. Create a `ServiceFeatureTable` using a feature service URL.
+2. Create a `FeatureLayer` from the service feature table.
+3. Create a `TextSymbol` to use for displaying the label text.
+4. Create a JSON string for the label definition.
     * Set the "LabelExpressionInfo.expression" key to express what the text the label should display. You can use fields of the feature by using `$feature.field_name` in the expression.
     * To use the text symbol, set the "symbol" key to the symbol's JSON representation using `textSymbol.toJson()`.
-4. Create a label definition from the JSON using `LabelDefinition.fromJson(json)`.`
-5. Add the definition to the feature layer with `featureLayer.getLabelDefinitions().add(labelDefinition)`.
-6. Lastly, enable labels on the layer using `featureLayer.setLabelsEnabled()`.
+5. Create a label definition from the JSON using `LabelDefinition.fromJson(json)`.
+6. Add the definition to the feature layer with `featureLayer.getLabelDefinitions().add(labelDefinition)`.
+7. Lastly, enable labels on the layer using `featureLayer.setLabelsEnabled()`.
 
 ## Relevant API
 
 * FeatureLayer
 * LabelDefinition
 * TextSymbol
+
+## About the data
+
+This sample uses the [USA 116th Congressional Districts](https://www.arcgis.com/home/item.html?id=cc6a869374434bee9fefad45e291b779) feature layer hosted on ArcGIS Online.
```

Comment:
our sample is currently using the 115th congressional districts. Either we should update the readme to reflect that, or better yet, update our sample so that we are using the common design data

Reply:
That's a really good spot. I've updated the sample data (incl. the code, to match the good practices we preach).

Reply:
oh cool, thank you!